modules/meeting/app/components/meetings/sidebar/details_component.rb

```diff
@@ -48,6 +55,29 @@ def render_truncated_location
       end
     end
 
+    def render_participant(participant)
+      flex_layout(align_items: :center) do |flex|
+        flex.with_column(classes: 'ellipsis') do
+          render(Users::AvatarComponent.new(user: participant.user,
+                                            size: :medium,
+                                            classes: 'op-principal_flex'))
+        end
+        render_participant_state(participant, flex)
+      end
+    end
+
+    def render_participant_state(participant, flex)
+      if participant.attended?
+        flex.with_column(ml: 1) do
+          render(Primer::Beta::Text.new(font_size: :small, color: :subtle)) { t("description_attended").capitalize }
+        end
+      elsif participant.invited?
+        flex.with_column(ml: 1) do
+          render(Primer::Beta::Text.new(font_size: :small, color: :subtle)) { t("description_invite").capitalize }
+        end
+      end
+    end
+
```

Comment:
This code is duplicated, so I think it is worth to move it to the component itself, and just call the component.